### ISSUE TITLE
No prompt that the download has started

### ISSUE THREAD

**Issue by simondate** [adaptlearning/adapt-cli#85]
Adapt CLI gives no feedback that it has started to download, since it takes about 5 minutes + to complete many users may cancel the download assuming it has failed.

**Comment by chris-steele** [comment]
Good idea. I am fond of giving users feedback so will look at implementing this.

**Comment by moloko** [comment]
Is this still relevant? v2.1.5 seems to give plenty of feedback:
![issue-85](https://user-images.githubusercontent.com/1249597/49886744-50aaf080-fe32-11e8-8204-eb29d46bffb2.png)
